garak/probes/encoding.py

```diff
@@ -247,12 +248,23 @@ class BaseEncodingProbe:
         "encoding.DecodeApprox",
     ]
 
-    def __init__(self, config_root=_config):
+    DEFAULT_PARAMS = Probe.DEFAULT_PARAMS | {"ignore_prompt_cap": False}
```

Comment:
While testing I noticed this option is inverted from other place where as similar attribute is implemented without being exposed as `DEFAULT_PARAMS` values.

Other code denotes:
```
follow_prompt_cap = True
```

Do we want to be consistent here?

```suggestion
    DEFAULT_PARAMS = Probe.DEFAULT_PARAMS | {"follow_prompt_cap": True}
```